detections/endpoint/secretdumps_offline_ntds_dumping_tool.yml

```diff
@@ -0,0 +1,51 @@
+name: SecretDumps Offline NTDS Dumping Tool
+id: 5672819c-be09-11eb-bbfb-acde48001122
+version: 1
+date: '2021-05-26'
+author: Teoderick Contreras, Splunk
+type: batch
+datamodel:
+- Endpoint
+description: This analytic detects a potential usage of secretsdump.py tool for dumping
+  credentials (ntlm hash) from a copy of ntds.dit and SAM.Security,SYSTEM registrry
+  hive. This technique was seen in some attacker that dump ntlm hashes offline after
+  having a copy of ntds.dit and SAM/SYSTEM/SECURITY registry hive.
```

Comment:
@t-contreras please see if you can add/reuse/extend this with the material we had before about secrets dumping in:

credential_dumping_via_copy_command_from_shadow_copy.yml
credential_dumping_via_symlink_to_shadow_copy.yml

ssa___assess_credential_strength_via_dsinternals_modules.yml
ssa___attempted_credential_dump_from_registry_via_reg_exe.yml
ssa___credential_extraction_dsinternals_conversion_modules.yml
ssa___credential_extraction_dsinternals_modules.yml
ssa___credential_extraction_fgdump_cachedump_s_option.yml
ssa___credential_extraction_fgdump_cachedump_v_option.yml
ssa___credential_extraction_getaddbaccount_from_dump.yml
ssa___credential_extraction_lazagne_command_options.yml
ssa___credential_extraction_mimikatz_modules.yml
ssa___credential_extraction_ms_debuggers_kernel_peek.yml
ssa___credential_extraction_ms_debuggers_z_option.yml
ssa___credential_extraction_powersploit_modules.ym
ssa___detect_dump_lsass_memory_using_comsvcs.yml

Reply:
Hi @miskoSplunk  thanks for this suggestion. I will check further on this and see if I can improve it. :)